### ISSUE TITLE
USER ISSUE: komm nicht ins Spiel, 

### ISSUE THREAD

**Issue by Greece62** [StrangeLoopGames/EcoIssues#25040]
\
**User Id: (51cf4bd8-10a5-4e20-76ab-08dc8d1bff58)**
**Version:** 0.11.0.0 beta release-693

**Steps to Reproduce:**
Neu auf Fehler durchgesucht

**Expected behavior:**
flüssiges Laden

**Actual behavior:**
Die folgende Fehlermeldung trat auf:
```
Vom Benutzer abgebrochen, da die Welt nicht hochgeladen wurde, nach langer Wartezeit. Immer bei 25 % stehengeblieben
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
Nein, keine Mods vorhanden

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by D3nnis3n** [comment]
Please provide the logfiles for client and server after trying to connect, as described here: https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-server-and-client-log-files-for-local-world

Also please note there is a known issue that reconnecting to a server is not currently possible - you need to restart the game, this will be addressed in the first hotfix.

**Comment by D3nnis3n** [comment]
This should be addressed with Hotfix 11.0.1, if the issue persists please create a new issue and attach your logfiles as noted in the issue template.
